adala/runtimes/_guidance.py

```diff
@@ -123,7 +118,7 @@ def record_to_record(
         instructions_template: str,
         output_template: str,
         extra_fields: Optional[Dict[str, Any]] = None,
-        field_schema: Optional[Dict] = None,
+        options: Optional[Dict] = None,
```

Comment:
I think field_schema had more clarity behind it. Do we have anything new in "options" other than field_schema?

Reply:
`field_schema` still remains as user facing parameter in skills. However, I'm unsure about what should be the proper runtime parameter, how customizable it has to be, why would we need broader definition of generations schema or is it too redundant, and only string input->string output is well enough for most of the use cases?